app/helpers/application_helper.rb

```diff
@@ -1,7 +1,7 @@
 module ApplicationHelper
   ### TEMP TEMP TEMP
   def videojs_rails(*_args)
-    content_tag(:p, 'Sorry, this video is unavailable', class: 'word_gloss', style: 'min-height: 150px')
+    content_tag(:p, t('Sorry, this video is unavailable'), class: 'word_gloss', style: 'min-height: 150px')
```

Comment:
Does this work? I think it's meant to be I18ln.t() -- or whatever the other parts of the app are doing.